pkg/logiface/logiface.go

```diff
@@ -0,0 +1,24 @@
+package logiface
+
+type Logger interface {
+	Errorf(format string, args ...interface{})
```

Comment:
small nit but modern code should use `any` over `interface{}` same in the other cases here.

Reply:
I will do this in a followup PR, left it because it was `nit`.